VERSION

```diff
@@ -1 +1 @@
-3.6.2
+3.7.0
```

Comment:
rule of thumb:
major 3.x.x: incompatible with previous versions
minor x.3.x: new features
maintenance x.x.3: bugfixes